x-pack/plugins/stack_alerts/server/rule_types/es_query/lib/fetch_search_source_query.ts

```diff
@@ -56,11 +56,10 @@ export async function fetchSearchSourceQuery({
   const { logger, searchSourceClient } = services;
   const isGroupAgg = isGroupAggregation(params.termField);
   const isCountAgg = isCountAggregation(params.aggType);
-
-  const initialSearchSource = await searchSourceClient.create(params.searchConfiguration);
+  const initialSearchSource = await searchSourceClient.createLazy(params.searchConfiguration);
 
   const index = initialSearchSource.getField('index') as DataView;
-  const { searchSource, filterToExcludeHitsFromPreviousRun } = updateSearchSource(
+  const { searchSource, filterToExcludeHitsFromPreviousRun } = await updateSearchSource(
```

Comment:
since we no longer use a DataViewLazy here, it's not necessary to change updateSearchSource to work async, since 

`const timeField = await index.getTimeField(); `

can be
`const timeField =index.getTimeField();`

This would lead to less code changes, side the dependent tests also don't need to be changed

Reply:
True, but this will use DataViewLazy shortly, so I left it.

Reply:
Ah I see, make sense then